app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -250,6 +253,14 @@ open class HomeActivity : LocaleAwareAppCompatActivity() {
                 ?.let { it as? TabTrayDialogFragment }
                 ?.also { it.dismissAllowingStateLoss() }
         }
+        // If there is a warm or hot startup, onNewIntent method is always called first.
```

Comment:
nit: newline above. You should space related code in blocks detached with newlines from other blocks

Reply:
done